### DISCUSSION TITLE
Creating a scope does not work

### DISCUSSION THREAD

**Discussion by Stefku** [LemonAppDev/konsist#770] [Ask a Question]
### Wanted end result.

Create a scope.

### What you tried so far.

I tried "Konsist.scopeFromProject()", but an exception occurred:
`com.lemonappdev.konsist.core.exception.KoInternalException: Project directory not found.`
I also tried scopeFromPackage.

In the docu I did not find a hint.

Any ideas?

**Comment by igorwojda** [comment]
Please attach sample project (best to take existing one remove all/most of the classes)

**Comment by igorwojda** [comment]
Since there is no followup I am unable to test this.
Perhaps will be fixed in next release by https://github.com/LemonAppDev/konsist/pull/831.
